SYCL/KernelAndProgram/spec_consts.cpp

```diff
@@ -2,7 +2,7 @@
 // Specialization constant values are not supported on CUDA
 // REQUIRES: opencl || level_zero
 // RUN: rm -rf %t/cache_dir
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %clangxx -fsycl -DSYCL_DEISABLE_FALLBACK_ASSERT=1 -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
There is a spelling mistake in the name of the `-D` option.

See also my comment in intel/llvm#3767 asking why we are disabling assertions in this test.

Reply:
Nice catch. Thanks!